modules/olm-installplan.adoc

```diff
@@ -5,4 +5,98 @@
 [id="olm-installplan_{context}"]
 = Install plan
 
-An _install plan_, defined by an `InstallPlan` object, describes a set of resources to be created to install or upgrade to a specific version of an Operator, as defined by a cluster service version (CSV).
+An _install plan_, defined by an `InstallPlan` object, describes a set of resources to be created by Operator Lifecycle Manager (OLM) for installing or upgrading to a specific version of an Operator, as defined by a cluster service version (CSV).
```

Comment:
Is it possible to avoid "to be created by?" "...describes a set of resources for the OLM to create for installling..."

If possible, can you break the sentence at the last Operator and then start a new one for the CSV.  I'm cowardly refusing a sugg because i don't know what has or reads the CSV.

Reply:
Attempt at a rework:

> A specific version of an Operator is defined by a `ClusterServiceVersion` (CSV) object. An _install plan_, defined by an `InstallPlan` object, describes a set of resources that Operator Lifecycle Manager (OLM) creates to install or upgrade to a given version of an Operator.